README.md

```diff
@@ -74,15 +86,10 @@ An input JSON file includes all genomic data files, input parameters and metadat
 
 ## How to organize outputs
 
-Install [Croo](https://github.com/ENCODE-DCC/croo#installation). Make sure that you have python3(> 3.4.1) installed on your system.
+Install [Croo](https://github.com/ENCODE-DCC/croo#installation). Make sure that you have python3(> 3.4.1) installed on your system. Find a `metadata.json` on Caper's output directory.
```

Comment:
A one liner explaining the importance of Croo will be great before this section.
It would be good to add a couple lines that- this step will be needed after the caper is successfully run. So, the user will know that this is not needed to run the actual pipeline.

Reply:
Something to "sell your product"- in the sense- on the lines of how good it will be to use Croo and not having to go through all the sub folders one by one to find all the output files.

Reply:
Also, some first time users may not know where to find the metadata.json. Some questions I would have had as a first time user will be- is this metadata.json same as input.json? I know the answer now but as a beginner, I would have not known it by reading this document. So it will be good to explain that as well.

Reply:
Also, I think it would be helpful to tell the user how to reach the metadata.json file- that is probably good to give an idea of the relative path where it is ideally located.

Reply:
sure, as i mentioned in your previous code review, i will make a new PR for this input/output things. I will leave this review as "unresolved" to track this conversation later for the new PR.